programs/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
-if (NOT WIN32)
+if (UNIX)
```

Comment:
On iOS:
> [316/316] Linking C executable programs/ziti-tunneler/ziti-tunneler-app.app/ziti-tunneler-app
FAILED: programs/ziti-tunneler/ziti-tunneler-app.app/ziti-tunneler-app
: && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -arch arm64 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.5.sdk -Wl,-headerpad_max_install_names -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS13.5.sdk programs/ziti-tunneler/CMakeFiles/ziti-tunneler-app.dir/ziti-tunneler.c.o  -o programs/ziti-tunneler/ziti-tunneler-app.app/ziti-tunneler-app -L/Users/dhart/repos/netfoundry/ziti-tunnel-apple/deps/ziti-tunneler-sdk-c/build-iphoneos-arm64/lib deps/ziti-sdk-c/library/libziti.a  lib/libziti_tunneler.a  lib/libziti_tunneler_cbs.a  lib/liblwipcore.a  lib/libziti_tunneler.a  lib/liblwipcore.a  lib/liblwipunixarch.a  deps/ziti-sdk-c/library/libziti.a  deps/ziti-sdk-c/deps/uv-mbed/libuv_mbed.a  deps/ziti-sdk-c/deps/uv-mbed/deps/libhttp-parser.a  deps/ziti-sdk-c/deps/uv-mbed/deps/libuv_link.a  deps/ziti-sdk-c/deps/uv-mbed/deps/libuv/libuv_a.a  -lpthread  deps/ziti-sdk-c/deps/uv-mbed/deps/mbedtls/library/libmbedtls.a  deps/ziti-sdk-c/deps/uv-mbed/deps/mbedtls/library/libmbedx509.a  deps/ziti-sdk-c/deps/uv-mbed/deps/mbedtls/crypto/library/libmbedcrypto.a  _deps/libsodium-build/lib/libsodium.a  -lm && :
Undefined symbols for architecture arm64:
  "_utun_open", referenced from:
      _main in ziti-tunneler.c.o
ld: symbol(s) not found for architecture arm64

Reply:
in ziti-sdk-c, excluded with:
> if(NOT ${PUB_OS} STREQUAL "ios" )
    add_subdirectory(programs)
endif()